Solutions/Recorded Future/Playbooks/RecordedFuture-DOMAIN-C2_DNS_Name-IndicatorProcessor/RecordedFuture-DOMAIN-C2_DNS_Name-IndicatorProcessor.json

```diff
@@ -125,7 +125,7 @@
                                             "domainName": "@{body('Parse_JSON')?['Name']}",
                                             "expirationDateTime": "@{addHours(utcNow(),2)}",
                                             "ingestedDateTime": "@{utcNow()}",
-                                            "targetProduct": "Microsoft Sentinel",
+                                            "targetProduct": "@{concat('Azure ', 'Sentinel')}",
```

Comment:
@RecordedFutureOskbo, there's no need to add concat we skipping the check for targetProduct, you can revert it to  'Azure Sentinel', thanks.